requirements.txt

```diff
@@ -3,4 +3,5 @@ isort
 types-decorator
 types-PyYAML
 types-setuptools
+mypy==1.5.1
```

Comment:
This is unrelated to the other changes I made.
Currently, tests do not pass on `master` branch with Mypy `1.6.0` and `1.6.1`. I could not figure out how to fix it.